src/cmd/run.go

```diff
@@ -279,6 +279,13 @@ func runCommand(container string,
 		"--workdir", workingDirectory,
 	}
 
+	// Podman sets keys 'ctrl-p' and 'ctrl-q' as the default detach keys. This behaviour is not desirable in
+	// Toolbox. Since Podman v1.8.2 it is possible to have set no detach keys by passing an empty string to the
+	// --detach-keys option. https://github.com/containers/podman/pull/5203
+	if podman.CheckVersion("1.8.2") {
```

Comment:
The Podman commits are such a mess. :(

The original issue was https://github.com/containers/podman/issues/4208 which led to https://github.com/containers/podman/pull/4441 However, then https://github.com/containers/podman/issues/5166 showed up which seems to say that https://github.com/containers/podman/pull/4441 didn't do what it claimed to do. So, finally we got https://github.com/containers/podman/pull/5203

Anyway, it seems to me that https://github.com/containers/podman/pull/5203 was already in 1.8.1.

Reply:
I updated the version to 1.8.1. I hope I got it right.

Reply:
Yes, it should be 1.8.1. 
See https://github.com/containers/podman/releases/tag/v1.8.1

Reply:
Thanks for confirming that, @junjieyuan